frontend/src/filter_table.tsx

```diff
@@ -72,74 +40,112 @@ const useStyles = makeStyles({
     color: 'blue',
     textDecoration: 'underline',
     cursor: 'pointer',
+  },
+  hidden: {
```

Comment:
What's happening here? Why not `display: none` or `visibility:hidden`?

Reply:
ah! forgot to change the name ^^ That class is for the sorting arrow.